syndeo-template/src/main/java/com/google/cloud/syndeo/SyndeoTemplate.java

```diff
@@ -189,11 +193,55 @@ public static ConfiguredSchemaTransform buildFromJsonConfig(JsonNode transformCo
                                         ? fieldNode.asText()
                                         : fieldNode.isArray()
                                             ? new ObjectMapper().convertValue(fieldNode, List.class)
-                                            : new ObjectMapper().convertValue(fieldNode, Map.class))
```

Comment:
you should address this later on, but it is also possible taht the field is a `Map` type. You will need to pass in the whole Field object (from [here](https://github.com/GoogleCloudPlatform/DataflowTemplates/pull/714/files#diff-a467001c5b58c9a73fafb006c786361ca4dad3e61c38bc8132503b3034b24b92R180)) to check if it's of type Map or type Row.